tests/test_telemetry.py

```diff
@@ -0,0 +1,144 @@
+    },
+    {
+      "status": {
+        "connectionLog": [
+          {
+            "date": "2022-05-09T15:54:35.121511+00:00",
+            "message": "Eof encountered"
+          },
+          {
+            "date": "2022-05-09T15:54:40.122113+00:00",
+            "message": "Eof encountered"
+          },
+          {
+            "date": "2022-05-09T15:54:45.122693+00:00",
+            "message": "Eof encountered"
+          }
+        ],
+        "connectionTime": None,
+        "lastTransmittedTime": "2022-05-09T15:54:41.128922+00:00",
+        "epoch": "2022-05-09T15:54:35.121228+00:00",
+        "connectionResetCount": 3,
+        "streamingEndpoint": {
+          "hostname": "100.123.0.8",
+          "protocol": "protoBufOverTcp",
+          "port": 64427
+        },
+        "dnsLog": [],
+        "connected": False,
+        "disconnectionTime": "2022-05-09T15:54:45.122685+00:00"
+      },
+      "streaming_type": "perfmon",
+      "sequencing_mode": "sequenced",
+      "protocol": "protoBufOverTcp",
+      "hostname": "100.123.0.8",
+      "port": 64427,
+      "id": "47c455f5-59ad-4582-9608-aa6c09398385"
+    },
+    {
+      "status": {
+        "connectionLog": [
+          {
+            "date": "2022-05-09T15:54:35.013046+00:00",
+            "message": "Eof encountered"
+          },
+          {
+            "date": "2022-05-09T15:54:40.013578+00:00",
+            "message": "Eof encountered"
+          },
+          {
+            "date": "2022-05-09T15:54:45.014154+00:00",
+            "message": "Eof encountered"
+          }
+        ],
+        "connectionTime": None,
+        "lastTransmittedTime": "2022-05-09T15:54:45.122669+00:00",
+        "epoch": "2022-05-09T15:54:35.012796+00:00",
+        "connectionResetCount": 3,
+        "streamingEndpoint": {
+          "hostname": "100.123.0.8",
+          "protocol": "protoBufOverTcp",
+          "port": 64428
+        },
+        "dnsLog": [],
+        "connected": False,
+        "disconnectionTime": "2022-05-09T15:54:45.014146+00:00"
+      },
+      "streaming_type": "events",
+      "sequencing_mode": "sequenced",
+      "protocol": "protoBufOverTcp",
+      "hostname": "100.123.0.8",
+      "port": 64428,
+      "id": "d8f5a012-232c-4aca-b5c9-637dbba8cb4b"
+    }
+  ]
+}
+
+def test_add_endpoint():
+    mgr.add_endpoint("fakehost", "fakeport", "faketype")
+    mock_rest.json_resp_post.assert_called_with(uri='/api/streaming-config', data={'hostname': 'fakehost', 'port': 'fakeport', 'streaming_type': 'faketype', 'sequencing_mode': 'sequenced', 'protocol': 'protoBufOverTcp'})
+
+def test_delete_endpoints():
+    mgr.delete_all_endpoints()
+    mock_rest.delete.assert_called()
+
+def test_get_endpoints(aos_api_version):
+    fixture_path = f"aos/{aos_api_version}/telemetry/endpoints.json"
+    mock_rest.json_resp_get.return_value = json.loads(read_fixture(fixture_path))
+    r = mgr.get_endpoints()
+    assert len(r) == 3
+    mock_rest.json_resp_get.assert_called()
+
```

Comment:
tox -e py38 flake8 fails multiple lines here. Please address and make sure it passes. 
```
flake8 run-test: commands[0] | flake8 aos/ tests/
tests/test_telemetry.py:2:1: F401 'telnetlib.AO' imported but unused
tests/test_telemetry.py:6:1: F401 'unittest.mock.call' imported but unused
tests/test_telemetry.py:9:1: F401 'tests.util.make_session' imported but unused
tests/test_telemetry.py:12:1: E302 expected 2 blank lines, found 0
tests/test_telemetry.py:130:1: E302 expected 2 blank lines, found 1
tests/test_telemetry.py:132:86: E501 line too long (220 > 85 characters)
tests/test_telemetry.py:134:1: E302 expected 2 blank lines, found 1
tests/test_telemetry.py:138:1: E302 expected 2 blank lines, found 1
tests/test_telemetry.py:144:1: W391 blank line at end of file
ERROR: InvocationError for command /Users/boothr/apstra-api-python/.tox/flake8/bin/flake8 aos/ tests/ (exited with code 1)
```

Reply:
Which has me curious why the GitHub actions to run these test don't fail. Ill dig into this.